spec/controllers/home_controller_spec.rb

```diff
@@ -74,8 +74,8 @@
 
       it 'filters out rentals not belonging to the current user' do
         get :index
-        expect(assigns[:rentals]).to include(create(:mock_rental, user_id: @user.id), create(:ongoing_rental, user_id: @user.id))
-        expect(assigns[:rentals]).not_to include(create(:mock_rental, user_id: @other_user.id))
+        expect(assigns[:rentals]).to include(create(:mock_rental, renter_id: @user.id), create(:ongoing_rental, renter_id: @user.id))
```

Comment:
I believe the designation is for a soft rental, one that does not reach out the api and a persistent rental that does contact the api.